### PR TITLE
feat: add post to slack

### PR DESCRIPTION
**Pull request by dgwhited**
Adds posting to a slack webhook url at the end.

### CODE REVIEW

File: kev_lambda.py [hunk 1]
```diff
@@ -83,5 +85,43 @@ def lambda_handler(event, context):
...
+                            "text": {"type": "plain_text", "text": tweet},
+                        }
+                    ]
+                }
+            ],
+        }
+        # Post to the slack channel
+        try:
+            requests.request(
+                "POST", url, headers=headers, data=json.dumps(payload).encode("utf-8")
```
Comment (willlindsey05) [inline kev_lambda.py:122]: `requests.post(url, json=payload)`

File: kev_lambda.py [hunk 2]
```diff
@@ -83,5 +85,43 @@ def lambda_handler(event, context):
...
+
+def post_to_slack(tweets):
+    """Posts to slack
+
+    Args:
+        tweets ([list]): Tweets to post to slack
+    """
+
+    # Set the headers for slack HTTP POST
+    headers = {"Content-Type": "application/json"}
```
Comment (willlindsey05) [inline kev_lambda.py:99]: This HTTP header is not needed , if you use. `requests.post(url, json=payload)` below

File: kev_lambda.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import boto3
 import requests
 import tweepy
+import os
```
Comment (willlindsey05) [inline kev_lambda.py]: Might not need os library.

File: kev_lambda.py [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
+import json
```
Comment (willlindsey05) [inline kev_lambda.py]: If you implement requests.post(url, json=payload), then I believe you can remove this library.

**Review by willlindsey05 — APPROVED** [review]

File: kev_lambda.py [hunk 5]
```diff
@@ -83,5 +83,41 @@ def lambda_handler(event, context):
...
+
+def post_to_slack(tweets):
+    """Posts to slack
+
+    Args:
+        tweets ([list]): Tweets to post to slack
+    """
+
+    # Set the headers for slack HTTP POST
+    headers = {"Content-Type": "application/json"}
```
Comment (dgwhited) [inline kev_lambda.py:99]:
```suggestion
```

**Review by willlindsey05 — APPROVED** [review]

**Pull request merged**